source/community/reactnative/src/adaptive-card.js

```diff
@@ -93,6 +95,8 @@ export default class AdaptiveCard extends React.Component {
 				payload: this.payload,
 				cardModel: this.cardModel
 			});
+            
+            this.scrollView.current.scrollTo({x: 0, y: 0, animated: false});
```

Comment:
@pavankumarj If the view to be scrolled only when the payload is updated, shall we add payload specific a condition here to avoid scrolling when other props like configManager are updated?

Reply:
Since we have defined the scrolling logic in componentDidUpdate and there is a anyways a check for updateKey, I don't think we need to add another condition, also most of the logic that deals with componentDidUpdate deals with payload which is why I have added scroll logic here. Thoughts?